apps/condo/domains/news/components/RecipientCounter.tsx

```diff
@@ -155,72 +136,62 @@ export const RecipientCounter: React.FC<RecipientCounterProps> = ({ newsItemScop
     const WillZeroReceiveHintMessage = intl.formatMessage({ id: 'news.component.RecipientCounter.willReceive.hintZero' })
 
     const [isXlsLoading, setIsXlsLoading] = useState(false)
+    const [isCountersLoading, setIsCountersLoading] = useState<boolean>(false)
+    const [counters, setCounters] = useState<{
+        propertiesCount: number,
+        unitsCount: number,
+        receiversCount: number
+    }>(null)
 
     const { organization } = useOrganization()
     const { push } = useRouter()
 
-    const propertyIdsFromNewsItemScopes = compact(uniq(map(newsItemScopes, 'property.id')))
+    const processedNewsItemScope = useMemo(() => {
+        return newsItemScopes.reduce<TNewsItemScopeNoInstance[]>((acc, scope) => {
+            const property = get(scope, 'property')
 
-    const { objs: properties, loading: loadingProperties } = Property.useAllObjects({
-        where: {
-            organization: { id: organization.id },
-            ...(propertyIdsFromNewsItemScopes.length > 1 ? { id_in: propertyIdsFromNewsItemScopes } : {}),
-            deletedAt: null,
-        },
-    })
-
-    const { objs: residents, loading: loadingResidents } = Resident.useAllObjects({
-        where: queryFindResidentsByOrganizationAndScopes(organization.id, newsItemScopes),
-    })
-
-    const processedNewsItemScope = newsItemScopes.reduce<TNewsItemScopeNoInstance[]>((acc, scope) => {
-        const partialScope: TNewsItemScopeNoInstance = {}
-        if (get(scope, 'property')) partialScope.property = scope.property
-        if (get(scope, 'unitType')) partialScope.unitType = scope.unitType
-        if (get(scope, 'unitName')) partialScope.unitName = scope.unitName
-        if (!isEmpty(partialScope)) acc.push(partialScope)
-        return [...acc]
-    }, [])
+            return [
+                ...acc,
+                {
+                    property: property ? { id: property.id } : null,
+                    unitType: get(scope, 'unitType'),
```

Comment:
Why not `null` by default. It would be nice to have same default value as `property` key.
```suggestion
                    unitType: get(scope, 'unitType', null),
```

Reply:
fixed